src/lang/lt-LT.json

```diff
@@ -82,85 +82,85 @@
   "label.is-not": "Is not",
   "label.is-not-set": "Is not set",
   "label.is-set": "Is set",
-  "label.join": "Join",
-  "label.join-team": "Join team",
-  "label.language": "Language",
+  "label.join": "Prisijungti",
+  "label.join-team": "Prisijungti į komandą",
+  "label.language": "Kalba",
   "label.languages": "Kalbos",
   "label.laptop": "Laptop",
   "label.last-days": "{x, plural, =0 {Paskutinės # dienų} zero {Paskutinės # dienų} one {Paskutinė diena} other {Paskutinės # dienos}}",
   "label.last-hours": "{x, plural, =0 {Paskutinės # valandų} zero {Paskutinės # valandų} one {Paskutinė # valanda} other {Paskutinės # valandos}}",
   "label.leave": "Leave",
-  "label.leave-team": "Leave team",
+  "label.leave-team": "Išeiti iš komandos",
   "label.less-than": "Less than",
   "label.less-than-equals": "Less than or equals",
   "label.login": "Prisijungti",
   "label.logout": "Atsijungti",
-  "label.manage": "Manage",
+  "label.manage": "Tvarkyti",
   "label.max": "Max",
-  "label.member": "Member",
-  "label.members": "Members",
+  "label.member": "Narys",
+  "label.members": "Nariai",
   "label.min": "Min",
   "label.mobile": "Mobilusis",
   "label.more": "Daugiau",
-  "label.my-account": "My account",
-  "label.my-websites": "My websites",
+  "label.my-account": "Mano paskyra",
+  "label.my-websites": "Mano svetainės",
   "label.name": "Pavadinimas",
   "label.new-password": "Naujas slaptažodis",
   "label.none": "None",
   "label.number-of-records": "{x} {x, plural, one {record} other {records}}",
   "label.ok": "OK",
-  "label.os": "OS",
-  "label.overview": "Overview",
+  "label.os": "Operacinės sistemos",
+  "label.overview": "Apžvalga",
   "label.owner": "Savininkas",
-  "label.page-of": "Page {current} of {total}",
+  "label.page-of": "Puslapis {current} iš {total}",
   "label.page-views": "Puslapių peržiūros",
-  "label.pageTitle": "Page title",
+  "label.pageTitle": "Puslapio pavadinimas",
   "label.pages": "Puslapiai",
   "label.password": "Slaptažodis",
   "label.powered-by": "Powered by {name}",
   "label.profile": "Profilis",
   "label.queries": "Queries",
-  "label.query": "Query",
-  "label.query-parameters": "Query parameters",
+  "label.query": "Užklausa",
+  "label.query-parameters": "Užklausų parametrai",
   "label.realtime": "Realiuoju laiku",
-  "label.referrer": "Referrer",
-  "label.referrers": "Referrers",
+  "label.referrer": "Persiuntėjas (referrer)",
+  "label.referrers": "Persiuntėjai (referrers)",
```

Comment:
the word "Persiuntėjas" isn't really common in LT lang, so I've kept the english one in `()` to clarify.
tho haven't tested in the UI if looks ok / no overflow